CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 --
 
+# 3.8.19
+
+## Added
+- Added `JEAGER_AGENT_HOST` and `JEAGER_AGENT_PORT` environment variable to allow specifying host and port separately instead of as the full URL.
```

Comment:
```suggestion
- Added `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT` environment variable to allow specifying host and port separately instead of as the full URL.
```

Reply:
🤦‍♂